homeassistant/components/config/entity_registry.py

```diff
@@ -114,9 +115,18 @@ async def websocket_update_entity(hass, connection, msg):
             websocket_api.error_message(msg["id"], "invalid_info", str(err))
         )
     else:
```

Comment:
Let's drop this `else` and instead do a `return` inside the `except`.

Reply:
Fixed.